docs/modules/demos/pages/data-lakehouse-iceberg-trino-spark.adoc

```diff
@@ -226,21 +235,21 @@ xref:nifi-kafka-druid-water-level-data.adoc#_nifi[nifi-kafka-druid-water-level-d
 https://spark.apache.org/docs/latest/structured-streaming-programming-guide.html[Spark Structured Streaming] is used to
 stream data from Kafka into the lakehouse.
 
-=== Accessing the Web Interface
+=== Accessing the web interface
```

Comment:
nitpick: We should keep the style of headlines consistent. It seems like you tried to remove all "ing" forms of verbs. If so, we also need to do that here.

Reply:
This would obviously also apply to all other headlines.

Reply:
Yeah, that got a little complicated, the whole _command_ (eg: Access) vs _suggestion_ (eg: Accessing, like how-to).

I treated the demos more like tutorials (sequence of steps to follow), so really they should be the command version.

At this point I think I'd rather just get in what I have done, and pick away at the other bits, as I still have some othre demo work to get through.